src/scss/libs/_normalize.scss

```diff
@@ -0,0 +1,84 @@
+@import '../variables';
+
+
+* {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  outline: none;
+  font-family: "Open Sans", sans-serif;
+  box-sizing: border-box;
+  -webkit-touch-callout: none; /* iOS Safari */
+  -webkit-user-select: none;   /* Chrome/Safari/Opera */
+  -moz-user-select: none;      /* Firefox */
```

Comment:
wtf is this ?

Reply:
https://developer.mozilla.org/ru/docs/Web/CSS/user-select